src/styles.ts

```diff
@@ -37,14 +37,14 @@ export const styles = css`
 		);
 		--contour-color: var(--line-color);
 		--contour-size: var(--cosmoz-input-contour-size);
+		--border-color: var(--contour-color);
+		--transform-y-percentage: var(--cosmoz-input-transform-y-percentage, 0%);
```

Comment:
I think that `--cosmoz-input-label-translate-y` would be a better name, because there is nothing stopping you from setting `--cosmoz-input-transform-y-percentage: 20px` and also you must specify that the label is the thing being translated.

Reply:
Fixed.